__tests__/users/App.test.js

```diff
@@ -72,6 +72,46 @@ describe('App Component', () => {
   afterAll(async () => {
     await browser.close();
   });
+  it('should fetch and append new users on subsequent pages for discord users tab when feature flag is on', async () => {
+    await page.goto(`${BASE_URL}/users/discord/?tab=in_discord&dev=true`);
+    await page.waitForNetworkIdle();
+
+    const userCardTestIds = await page.$$eval(
+      '[data-testid^="user-card-"]',
+      (cards) => cards.map((card) => card.getAttribute('data-testid')),
+    );
+    expect(userCardTestIds.length).toBe(2);
+  });
+  it('should fetch and append new users on subsequent pages for discord users tab', async () => {
+    await page.goto(`${BASE_URL}/users/discord/?tab=in_discord`);
+    await page.waitForNetworkIdle();
+
+    const userCardTestIds = await page.$$eval(
+      '[data-testid^="user-card-"]',
+      (cards) => cards.map((card) => card.getAttribute('data-testid')),
+    );
+    expect(userCardTestIds.length).toBe(10);
+  });
```

Comment:
I am not sure how will this test will pass. Can you please attach the test result of this file

Reply:
Sorry had not pushed my changes, updated it now

Reply:
@IshanVeer If the comments are resolve please mark them resolved.